andela/travel_tool_back
# #161477594 Allow addition and deletion of rooms while editing guest house

#### What does this PR do?

Allow Travel Admin to be to add and remove room/s while editing guest house

#### Description of Task to be completed?

- Fetch data from frontend and generate beds for the added rooms. The existing rooms that have beds are not modified in this case while those that have been deleted are not removed. 
- The task makes sure that while editing guest house, the travel admin can add or delete room/s. 

#### How should this be manually tested?
- git clone the repo and cd to the repo
- git checkout bg-fix-editing-guest-house-161477594
- run these commands: 
   - make ssh
   - make stop 
   - make clean 
   - make rollback
   - make migrate
   -  make seed
   -  make start
  - NB: Make sure that when running make rollback/migrate/seed, make start should be running.
- in the frontend of the application, checkout to develop
- run, the frontend app and add guest house. Make sure you are travel admin
- Try edit the added guest house by adding a room, then after success, try edit the room again by delteting added room. 
#### Any background context you want to provide?
- No branch for this PR for frontend. Use develop instead. 
#### What are the relevant pivotal tracker stories?

[#161477594](https://www.pivotaltracker.com/story/show/161477594)

#### Screenshots (if appropriate)
<img width="1432" alt="screen shot 2018-11-01 at 15 04 36" src="https://user-images.githubusercontent.com/32410518/47850787-7ed6f280-dde7-11e8-9159-c324b7700b0d.png">

#### Questions:
- None
